src/v3/src/transformer/layout/email/transformEmailAuthenticatorEnroll.ts

```diff
@@ -74,6 +74,7 @@ export const transformEmailAuthenticatorEnroll: IdxStepTransformer = ({ transact
       : undefined;
     subTitleElement.options.content = getEmailAuthenticatorSubtitle(
       emailAddress,
+      undefined,
```

Comment:
This param is always `undefined`?

Reply:
Yeah. Backend won't fill in secondary email details during enrollment. It's only required during verification step. In V3, the code was being shared between both flows.